staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto

```diff
@@ -731,6 +731,28 @@ message OwnerReference {
   optional bool blockOwnerDeletion = 7;
 }
 
+// PartialObjectMetadata is a generic representation of any object with ObjectMeta. It allows clients
```

Comment:
same, this got fixed in https://github.com/kubernetes/kubernetes/pull/77403, it shouldn't be in your PR